standard/sections/clause_7_normative_text.adoc

```diff
@@ -406,10 +402,10 @@ include::../recommendations/core/REC_extent_temporal.adoc[]
 "time": null
 ----
 
```

Comment:
Additional example with two repeating intervals:

```
[source, json]
----
"time": {
    "interval": [
        ["R/T00Z", "PT180H"],
        ["R/T12Z", "PT180H"]
    ],
    "resolution": "PT6H"
}
----

```

Feel free to pretty print and move around the example as needed :)